content/jobs.rst

```diff
@@ -0,0 +1,60 @@
+public: no
+headline:
+  - type: 'Now Hiring'
+    tagline: collaborative team, flexible hours, competitive pay
+summary: |
+  OddBird is a small team (you'll be #6),
+  working on a mix of client and internal projects.
+  This is a contract job with long term potential,
+  a good hourly rate (equal across the team),
+  and a lot of flexibility.
+
+  *We'd love to hear from you!*
+
+
+JavaScript Web Developer
+========================
+
+OddBirds:
+---------
+
+- Enjoy working collaboratively on complex problems
+- Are involved with the vision and direction of the company:
+  how we work, who we work for, and how we manage resources
+- Are paid a competitive hourly rate,
+  equally across the entire team
+- Write tests (100% coverage everywhere, always) and documentation
+- Give and receive code reviews as collaborators
+- Are encouraged to have lives & interests outside of work
+- Set our own hours, and work from anywhere
+- Listen to users, and enjoy having our assumptions challenged/tested
+- Care about diversity in the tech community
+- Engage with the software community as part of the job –
+  writing articles, sharing tools, and giving talks
+
+We work together to find clients,
+and we work together to build them the best tools we can.
+When there's work to be done, we all do what we can,
+and we get paid for our time.
+
+You:
+----
+
+- Have experience writing JavaScript for the web,
+  preferably not limited to a single framework
+- Enjoy solving complex problems with readable, efficient, tested code
+- Can explain the decisions behind your work, and the tradeoffs involved
+- Work closely with designers to integrate form & function
+- Enjoy the push-and-pull of a tight feedback cycle
+- Enjoy exploring outside what you know, and questioning your assumptions
+
+We’re looking for 10-20 billable hours per week to start,
+with potential for more.
+@@@ Something tho express that "full time" is ~25hrs,
+this is not as half-time as it looks?
+
+Our process keeps design and development tightly integrated.
+There are no Photoshop comps thrown over the wall to developers.
+Everyone is working together (with clients and users) towards a unified goal.
+While we are specifically hiring for JavaScript experience,
+we’re interested in the full range of your skills and interests.
```

Comment:
Do we want a clearer CTA at the end of this post? The default footer one is targeted toward potential clients, which might not fit this page.